wiki/Tournaments/Official_support/en.md

```diff
@@ -0,0 +1,181 @@
+
+- The name of your tournament
+- A brief description of your tournament, including its scoring type and any other appropriate features
+- Any applicable dates for your tournament
+- Any links to public off-site chatrooms or servers used for the event (Discord, etc.)
+- A link to the forum thread located under the [Tournaments subforum](https://osu.ppy.sh/community/forums/55) that announces/details your tournament
+- Any links to previous iterations of the same tournament (only where applicable)
+- A list of the players involved in running the tournament
+- A list of players who have registered for the tournament
+
+Please allow **at least a week** for a response. We will try our best to handle all requests within a week, but we occasionally have large spikes in workload that can make this impossible.
+
+### Criteria
+
+Community-run tournaments which abide by the following hard criteria are eligible for support:
+
+- The tournament must have at least 64 players or 16 teams minimum into the qualifier rounds, **OR** operate on a Round of 16 double elimination at a minimum. Larger scales and formats are permitted.
+- The tournament is organized and run by an *experienced team of accomplished and reputable volunteers***¹** , **OR**, the tournament *has been run at least once before without receiving rewards*.
+- Any of the staff members involved in the managing or organization of the tournament must not be participating in the tournament themselves at any point.
+    - To clarify further, if you play in a given tournament at all, you are unable to later assist as a referee, mappool selector or anything similar until that instance of the tournament concludes. You either play or help run, not both.
+- The tournament must be run only four or fewer times per year.
+- The tournament must have all players screened by the [account support team](/wiki/People/The_Team/Account_support_team) **before play commences**.
+- The tournament must not be "self-reffed". A dedicated referee must be present during every match.
+- The tournament must not prevent players, who pass the screening, from participation without both ample evidence presented publicly against them and the approval of the account support team. 
+    - This only applies in circumstances where they would otherwise meet the sign-up criteria for the tournament.
+    - This includes preventing users who are perceived to be "sandbagging" from play.
+        - Should an organizer have valid concerns about the presence of such players affecting the competitive integrity of their tournament, they may raise the issue to the Tournament Committee for a case-by-case review using the link listed below.
+- The forum thread for the tournament must contain clearly visible link in normal-sized font to [this reporting form](https://pif.ephemeral.ink/tournament-reports) as the very last content of the original post.
+    - ```https://pif.ephemeral.ink/tournament-reports```
```

Comment:
```suggestion
- The tournament must have at least 64 players or 16 teams minimum into the qualifier rounds, **OR** operate on a Round of 16 double elimination at a minimum. Larger scales and formats are permitted.
- The tournament is organized and run by an *experienced team of accomplished and reputable volunteers***¹** , **OR**, the tournament *has been run at least once before without receiving rewards*.
- Any of the staff members involved in the managing or organization of the tournament must not be participating in the tournament themselves at any point.
  - To clarify further, if you play in a given tournament at all, you are unable to later assist as a referee, mappool selector or anything similar until that instance of the tournament concludes. You either play or help run, not both.
- The tournament must be run only four or fewer times per year.
- The tournament must have all players screened by the [account support team](/wiki/People/The_Team/Account_support_team) **before play commences**.
- The tournament must not be "self-reffed". A dedicated referee must be present during every match.
- The tournament must not prevent players, who pass the screening, from participation without both ample evidence presented publicly against them and the approval of the account support team. 
  - This only applies in circumstances where they would otherwise meet the sign-up criteria for the tournament.
  - This includes preventing users who are perceived to be "sandbagging" from play.
    - Should an organizer have valid concerns about the presence of such players affecting the competitive integrity of their tournament, they may raise the issue to the Tournament Committee for a case-by-case review using the link listed below.
- The forum thread for the tournament must contain clearly visible link in normal-sized font to [this reporting form](https://pif.ephemeral.ink/tournament-reports) as the very last content of the original post.
  - ```https://pif.ephemeral.ink/tournament-reports```
```